src/rust/grapl-crypto/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+pub(crate) mod hasher;
+pub mod aead;
+pub mod asymmetric;
+pub mod rand_bytes;
+
+// Lets us serialize/ deserialize arrays > 32 bytes
+mod arrays {
```

Comment:
_Definitely_ need some tests on this

Reply:
Also, it seems a bit odd to embed this module in `lib.rs` directly; can we move it to its own file?